tracer/test/Datadog.Trace.Tools.dd_dotnet.ArtifactTests/RunCommandTests.cs

```diff
@@ -0,0 +1,112 @@
+// <copyright file="RunCommandTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Threading.Tasks;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.Tools.dd_dotnet.ArtifactTests;
+
+public class RunCommandTests : ConsoleTestHelper
+{
+    public RunCommandTests(ITestOutputHelper output)
+        : base(output)
+    {
+    }
+
+#if NETFRAMEWORK
+    [SkippableFact]
+    [Trait("RunOnWindows", "True")]
+    public async Task Run()
+    {
+        // This test is only for .NET Framework because it uses a syntax that doesn't allow to pass arguments to the target application
+        // So it won't work with samples that must be started as dotnet <sampleName>
```

Comment:
Maybe a silly question, but why can't we use a syntax here that _does_ allow parsing arguments to the target application, like you do in the next test?

Reply:
Basically there are two syntaxes:

`dd-dotnet run myApp` => with this one, we wan't give arguments to myApp (because they will be interpreted as arguments for `dd-dotnet run`
`dd-dotnet run -- myApp` => here, we can give arguments to myApp

This test validates that the first syntax works. However, on .NET Core we launch the samples by running `dotnet myApp.dll` so I *need* an argument (and therefore can't test the case where there is no arguments)

Reply:
oh I see, you're explicitly testing that example, fair enough. 

FWIW, you _could_ launch the sample on Windows using the exe that's created, no? But meh